src/cluster_cf.yaml

```diff
@@ -284,15 +284,14 @@ Resources:
       - arn:aws:iam::aws:policy/AmazonEKSClusterPolicy
       - arn:aws:iam::aws:policy/AmazonEKSServicePolicy
       RoleName: !Sub '${AWS::StackName}-eks-control-plane-role'
-  
+
   EksCluster:
     Type: "AWS::EKS::Cluster"
-    DependsOn: 
+    DependsOn:
     - EksControlPlaneRole
     - ControlPlaneSecurityGroup
     Properties:
       Name: !Sub "umsi-easy-hub-${Tag}-EksCluster"
-      Version: "1.12"
```

Comment:
OK, which version do you have in your deployment? I think it is actually nice to have the version pinned (but the latest stable, not the old 1.12, of course).

Reply:
Mine got created with the current, 1.19. Not sure if you used latest when making si330, but that one's already 2 versions old at 1.17

Reply:
OK, let's pin on 1.19... otherwise, we have a huge "moving" piece.